spec/locales-currencies-tz.html

```diff
@@ -21,7 +21,7 @@ <h1>Case Sensitivity and Case Mapping</h1>
     <h1>Language Tags</h1>
 
     <p>
-      The ECMAScript 2019 Internationalization API Specification identifies locales using language tags as defined by IETF BCP 47 (RFCs 5646 and 4647 or their successors), which may include extensions such as those registered through RFC 6067. Their canonical form is specified in RFC 5646 section 4.5 or its successor.
+      The ECMAScript 2019 Internationalization API Specification identifies locales using language tags as defined by IETF BCP 47 (RFCs 5646 and 4647 or their successors), which may include extensions such as those registered through RFC 6067. Their canonical form is that of a Unicode BCP 47 Locale Identifier, as specified in <a href="http://unicode.org/reports/tr35/#BCP_47_Conformance">Unicode Technical Standard #35 LDML § 3.3 BCP 47 Conformance</a>.
     </p>
 
     <p>
```

Comment:
Suggested Changes in IsStructurallyValidLanguageTag

```
  represents a well-formed 
- BCP 47 language tag as specified in RFC 5646 section 2.1,
+ Unicode BCP 47 Locale Identifier" as specified in Unicode Technical Standard 35 section 3.2,
  or successor,
```

Reply:
Done